src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx

```diff
@@ -68,6 +79,13 @@ export default function PageFooter(props: Props) {
           "Upload"
         )}
       </Button>
+      <Checkbox
```

Comment:
This should be centered within the container (the checkbox should also be centered to the text)

Reply:
resolved in [4e933f6](https://github.com/aics-int/aics-file-upload-app/pull/116/commits/4e933f6acba2f622da8d9fe9a79869cc552ee5a0)